.github/workflows/push_pull_request_test.yml

```diff
@@ -11,6 +11,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Change dir owner to working user
+        run: sudo chown -R $USER:$USER $GITHUB_WORKSPACE
```

Comment:
```suggestion
```

Reply:
I tried changing the user to fix the CI Permission Error when creating and deleting files in tests, but it didn't work. It's best to delete this.